c/include/libsbp/cpp/state.h

```diff
@@ -68,12 +68,12 @@ class State {
     return sbp_process(&state_, &read_func);
   }
 
-  s8 send_message(u16 msg_type, u16 sender_id, u8 length, u8 payload[]) {
-    return sbp_send_message(&state_, msg_type, sender_id, length, payload, &write_func);
+  s8 send_message(u16 msg_type, u16 sender_id, u8 length, const u8 payload[]) {
+    return sbp_send_message(&state_, msg_type, sender_id, length, const_cast<u8 *>(payload), &write_func);
```

Comment:
any chance you can update the sbp library to avoid this `const_cast`?

Reply:
It would be a major breaking change. When we have a need to revamp the library it can be considered then.